test-requirements.txt

```diff
@@ -0,0 +1,7 @@
+flake8==2.4.1
+mccabe==0.3.1
+mock==1.3.0
+nose==1.3.7
```

Comment:
we shouldn't need nose and pytest. I think talking with @markramm the thought is to stick to pytest.